app/javascript/mastodon/features/report/category.js

```diff
@@ -42,7 +42,7 @@ class Category extends React.PureComponent {
 
     switch(category) {
     case 'dislike':
-      onNextStep('thanks');
```

Comment:
Hard to say what the implications are. But let's try in stage.

Reply:
I tested locally, it seems to just allow you to submit the report with the correct (`dislike`) category, but I'd like to make sure the report looks OK in stage (I couldn't test that part locally)